DataFormats/Portable/interface/HostProduct.h

```diff
@@ -0,0 +1,31 @@
+#ifndef DataFormats_Common_interface_HostProduct_H
+#define DataFormats_Common_interface_HostProduct_H
+
+#include <alpaka/alpaka.hpp>
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+// a heterogeneous unique pointer...
+template <typename T>
+class HostProduct {
```

Comment:
Is this a temporary measure to help the transition, or intended for long term?

Reply:
Correct, it's provisional.